### PR TITLE
refactor: use simple babel config and drop babel runtime

### PR DESCRIPTION
**Pull request by TrySound**
**What**:

With modern node there is no need for transpilation (except esm to commonjs)
and having babel/runtime dependency gives nothing as it's not used by
generated code as well. In this diff I replaced kcd-scripts babel config with simple
commons plugin for tests. Published code stays the same and unsupported
node versions should be caught by tests. Babel runtime is not necessary
anymore as dependency.

**Why**:

**How**:

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged

### CODE REVIEW

**Comment by TrySound** [comment]
cc @conartist6

**Comment by conartist6** [comment]
Remove the module transform plugin too and all of babel. You're not using the files it's generating. I'd remove the build script as well (if it no longer does anything useful) and remove the gitignore and any other ignores on `dist`.

**Comment by TrySound** [comment]
Don't see transform-runtime plugin though found commonjs pluginn from babel6

File: package.json [hunk 1]
```diff
@@ -49,8 +48,8 @@
     "dist"
   ],
   "babel": {
-    "presets": [
-      "./other/babel-config.js"
+    "plugins": [
+      "@babel/transform-modules-commonjs"
```
Comment (conartist6) [inline package.json:52]: Why do we need this? What is using it?

Comment (TrySound) [reply package.json:52]: Jest requires commonjs transform

**Pull request merged**